state_machine/drivers/emulation/lib/pycubed.py

```diff
@@ -123,14 +135,24 @@ def log(self, str):
         str = (str[:20] + '...') if len(str) > 23 else str
         print(f'log not implemented, tried to log: {str}')
 
+    @property
+    def imu(self):
+        return True
+
     @property
     def sun_vector(self):
-        """Returns the sun pointing vector in the body frame"""
-        return array([0, 0, 0])
+        """returns the sun pointing vector in the body frame"""
+        return array([self._luxp[0] - self._luxn[0],
+                      self._luxp[1] - self._luxn[1],
+                      self._luxp[2] - self._luxn[2]])
 
     @property
-    def imu(self):
-        return True
+    def lux_p(self):
```

Comment:
I'm not seeing how these functions (lux_p, and lux_n) add anything useful to what we already have in pycubed.py, let's leave them out.